CMakeLists.txt

```diff
@@ -0,0 +1,52 @@
+cmake_minimum_required(VERSION 2.8.12)
+
+project(RenderDoc CXX C)
+
+if(WIN32)
+    message(FATAL_ERROR "No Win32 support")
```

Comment:
Can you make this error more descriptive just in case someone tries to run cmake on windows? e.g.

"CMake is not needed on windows, just open and build renderdoc.sln"